TCCE 4.5 Release Push

GlacierPark19 opened the pull request:
No clue what to title. lol

If there are issues, please let me know. So exciting!

Chappiem72 committed:
Added additional Creative tab

Chappiem72 committed:
Added additional Creative Tabs

Chappiem72 committed:
More teweaks!

Chappiem72 committed:
Breaking things apart. Whoot@

Chappiem72 committed:
Seperating Materials now

Chappiem72 committed:
disabled Zepplin in config by default

Chappiem72 committed:
Renamed that Painbrush!

Chappiem72 committed:
Removed WigWag

Chappiem72 committed:
More wigwag removal

Chappiem72 committed:
Testing

Chappiem72 committed:
Distill fixes

Chappiem72 committed:
Fixes for the SNCB_M6. Thanks to TheDoctor1138

Chappiem72 committed:
Fixed Distill change that I made

Chappiem72 committed:
Creative Tab Changes

Chappiem72 committed:
Fixed some silly/stupid formating errors

GlacierPark19 committed:
Fixed some silly/stupid formating errors

GlacierPark19 committed:
More Fixes!

GlacierPark19 committed:
Removed tooManyTabs

GlacierPark19 committed:
Delete .theia directory

GlacierPark19 committed:
Delete .vscode directory

GlacierPark19 committed:
The WigWag is back!

GlacierPark19 committed:
Final updates for 4.3 release

GlacierPark19 committed:
Fixes from Chazi898!

GlacierPark19 committed:
Merge branch '1.7-Community-4.3' of https://github.com/GlacierPark19/Traincraft-5 into 1.7-Community-4.3

Chappiem72 committed:
!!PLEASE READ!! ALL of WWCP's CONTENT HAS BEEN REMOVED.
PLEASE USE THIS PUSH

Chappiem72 committed:
Merge branch '1.7-Community-4.3' of https://github.com/GlacierPark19/Traincraft-5 into 1.7-Community-4.3

Chappiem72 committed:
Changes made to make this version my own

GlacierPark19 committed:
New Content??

GlacierPark19 committed:
Merge branch '1.7-Community-4.3' of https://github.com/GlacierPark19/Traincraft-5 into 1.7-Community-4.3

GlacierPark19 committed:
New Content??

GlacierPark19 committed:
Rendering attempts

GlacierPark19 committed:
MODELS WORK!

Figured out rendering finally. Here we go! Home stretch?

GlacierPark19 committed:
Adding and removing some things to fix an error

GlacierPark19 committed:
BNLRV A unit!

I'm finally getting the hang of this! WHOOT

GlacierPark19 committed:
Changes for "versionization".

Changed Mod Version
Lang fixes
Soul removal

GlacierPark19 committed:
BNLRV Works!

Finnaly!

GlacierPark19 committed:
Work for new content+bug fixes

GlacierPark19 committed:
Lots of New Content. If there are bugs Please discord me!

I See you TheDoctor

GlacierPark19 committed:
More Content Additions!

GlacierPark19 committed:
Some Final Fixes before TCCE 5's release.

GlacierPark19 committed:
Some Final Fixes before TCCE 4.5's release.

It's ready

EternalBlueFlame commented:
This push contains file conflicts from not pulling before you pushed.
Additionally you should finish ConfigHandler line 80.
Also vor versioning and compatibility reasons, revert the build.gradle to using config.modVersion.toString() and revert the mcmodinfo back to using the
  "version": "${version}",
  "mcversion": "${mcversion}",

GlacierPark19 commented:
Made some changes: 

Removed the tab split. It's unnecessary. I need to do some reading into the best way to implement it. I may re add it in the future
Fixed the mod version and gradle issues
